lib/cli/commands/channelbalance.ts

```diff
@@ -1,6 +1,39 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ChannelBalanceRequest } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { ChannelBalanceRequest, ChannelBalanceResponse } from '../../proto/xudrpc_pb';
+
+const HEADERS = [
+  colors.red('Ticker'),
+  colors.red('balance'),
```

Comment:
This should probably be capital B "Balance". Also are we ok with sticking to a blue color for headers to distinguish them from the sell & buy header colors? If so can we change that here as well as the other headers added in this PR?